src/component/Calculator.js

```diff
@@ -1,30 +1,190 @@
-import React from 'react';
+import React, { Component } from 'react';
+import calculate from '../logic/calculate';
 
-function Calculator() {
-  return (
-    <div className="main">
-      <div className="screen">0</div>
-      <div className="btn light-gray">AC</div>
-      <div className="btn light-gray">+/-</div>
-      <div className="btn light-gray">%</div>
-      <div className="btn org">÷</div>
-      <div className="btn">7</div>
-      <div className="btn">8</div>
-      <div className="btn">9</div>
-      <div className="btn org">X</div>
-      <div className="btn">4</div>
-      <div className="btn">5</div>
-      <div className="btn">6</div>
-      <div className="btn org">-</div>
-      <div className="btn">1</div>
-      <div className="btn">2</div>
-      <div className="btn">3</div>
-      <div className="btn org">+</div>
-      <div className="btn zero">0</div>
-      <div className="btn">.</div>
-      <div className="btn org">=</div>
-    </div>
-  );
+// eslint-disable-next-line react/prefer-stateless-function
+/* eslint-disable react/destructuring-assignment */
```

Comment:
Kindly avoid disabling linter checks inline. They represent coding standards that you share with your team and most of the time enforce best coding practices.